moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -499,7 +499,15 @@ KinematicState extractRobotState(const moveit::core::RobotStatePtr& robot_state,
   current_state.joint_names = joint_names;
   robot_state->copyJointGroupPositions(joint_model_group, current_state.positions);
   robot_state->copyJointGroupVelocities(joint_model_group, current_state.velocities);
+
   robot_state->copyJointGroupAccelerations(joint_model_group, current_state.accelerations);
+  // If any acceleration is nan, set all accelerations to zero
```

Comment:
It feels weird to me to have to do a nan check here.

If this is specifically to protect from output that the Ruckig plugin could provide in the event of large acceleration limits, it should probably be handled in the smoothing plugin itself.

But if you noticed this happening with other implementations, that could change things.

Reply:
This appears to be a bug deep within MoveIt that the first RobotState has `nan` accelerations. I didn't want to add this check but it was necessary.

I guess I'll add a TODO here and make an issue to fix the root cause.